src/Hypernode/Magento/Command/Hypernode/Performance/PerformanceCommand.php

```diff
@@ -88,6 +88,12 @@ protected function execute(InputInterface $input, OutputInterface $output)
             $this->_options['compare-url'] = $this->getEffectiveUrl($this->_options['compare-url']);
         }
 
```

Comment:
it would be better to decouple the sitemap parsing from the path detection. suggestion by @tdgroot:
```
Timon de Groot [10:30 AM] 
Try catch kan ook idd
[10:33] 
Om dit script netjes te maken
[10:33] 
Zou je het in wat meer classes moeten zetten
[10:33] 
Dus eigenlijk 1 service class waar alle logica zit
[10:33] 
En 2 classes waar alle input geparsed wordt
[10:34] 
1 voor M1 en 1 voor M2
[10:34] 
Die sturen de input dan weer door naar de service class
```